driver/pace/driver/run.py

```diff
@@ -1,181 +1,179 @@
-import os
-import sys
-from dataclasses import fields
-from typing import List, Optional, Tuple
+import abc
+import dataclasses
+import functools
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, Tuple, Union
 
 import click
-import f90nml
-import numpy as np
+import dacite
+import yaml
+import zarr.storage
 from mpi4py import MPI
-from typing_extensions import Literal
 
+import fv3core
 import fv3core.initialization.baroclinic as baroclinic_init
```

Comment:
[not for this PR]: but we should move this function to driver too right?

Reply:
Possibly, possibly not. I believe this function is unit tested in fv3core? There are some reasons to want to be able to initialize a DycoreState even if you don't have the `driver` package. It doesn't depend on any non-dycore packages, and conceptually it's very coupled to `DycoreState`, so it's probably fine for it to live in fv3core.

If this function also initialized physics state, it would need to be in `driver`.